### PR TITLE
Add DB script changes related to shared scopes

### PR DESCRIPTION
**Pull request by dushaniw**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/db2.sql [hunk 1]
```diff
@@ -2018,6 +2017,23 @@ CREATE TABLE AM_API_URL_MAPPING (
...
 
+CREATE TABLE AM_API_RESOURCE_SCOPE_MAPPING (
+            SCOPE_NAME VARCHAR(255) NOT NULL,
+            URL_MAPPING_ID INTEGER NOT NULL,
+            TENANT_ID INTEGER NOT NULL,
+            FOREIGN KEY (URL_MAPPING_ID) REFERENCES AM_API_URL_MAPPING(URL_MAPPING_ID) ON DELETE CASCADE,
+            PRIMARY KEY(SCOPE_NAME, URL_MAPPING_ID)
+)/
+
+CREATE TABLE AM_KM_MGT_APPLICATION (
```
Comment (tharindu1st) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/db2.sql:2028]: remove this

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mssql.sql [hunk 2]
```diff
@@ -1439,6 +1438,27 @@ CREATE TABLE AM_API_URL_MAPPING (
...
+    SCOPE_NAME VARCHAR(255) NOT NULL,
+    URL_MAPPING_ID INTEGER NOT NULL,
+    TENANT_ID INTEGER NOT NULL,
+    FOREIGN KEY (URL_MAPPING_ID) REFERENCES AM_API_URL_MAPPING(URL_MAPPING_ID) ON DELETE CASCADE,
+    PRIMARY KEY(SCOPE_NAME, URL_MAPPING_ID)
+);
+
+IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[AM_KM_MGT_APPLICATION]') AND TYPE IN (N'U'))
+
+CREATE TABLE AM_KM_MGT_APPLICATION (
```
Comment (tharindu1st) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mssql.sql:1453]: same goes here

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/db2.sql (hunk 1 above)
Comment (dushaniw) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/db2.sql]: done

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mssql.sql (hunk 2 above)
Comment (dushaniw) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mssql.sql]: done

**Pull request merged**